src/Analysis/Engine/Impl/Values/SpecializedNamespace.cs

```diff
@@ -208,52 +209,17 @@ abstract class SpecializedNamespace : AnalysisValue {
             return _original.IsOfType(klass);
         }
 
-        public override IEnumerable<ILocationInfo> Locations {
-            get {
-                if (_original == null) {
-                    return new LocationInfo[0];
-                }
-                return _original.Locations;
-            }
-        }
+        public override IEnumerable<ILocationInfo> Locations => _original.Locations?.MaybeEnumerate();
```

Comment:
Original check was about `_original` field being `null`, not its `Locations` property. Is that an expected change?

Reply:
Fixed